pkg/controller/redirect/helpers.go

```diff
@@ -15,10 +15,30 @@
 package redirect
 
 import (
+	"net"
+	"net/http"
 	"net/url"
 	"strings"
 )
 
+func HostRegionFromRequest(r *http.Request, hostnameToRegion map[string]string) string {
```

Comment:
Doc

Reply:
removed - this ended up being dead code.